oeps/processes/oep-0055-project-maintainers.rst

```diff
@@ -0,0 +1,345 @@
+
+Specification
+=============
+
+Definitions
+-----------
+
+The term **repository** will mean a repository on the GitHub social
+coding platform.
+
+The term **component** will typically mean the unit of software housed
+in a single GitHub repository.  Exceptions will be made in the case of
+edx-platform where there may be multiple distinguishable components in
+that single repository.  Components will typically, but not always, be
+hosted at the `openedx GitHub organization
+<https://github.com/openedx/https://github.com/openedx/>`_.
+Components will be "part of" or "critical to" the open-source
+platform.
+
+The term **maintainer** will mean a named person or persons
+responsible for the health of component and the component's community.
+The remainder of this specification will be focused on the details of
+the role.
+
+The term **core contributor** will mean a named person participating
+in the `Core Contributor Program`_ as a coding core contributor and
+meeting the requirements of that program.
+
+The term **maintainer scrum-of-scrums** will mean group of all
+maintainers who are responsible for flagging issues to project
+leadership and proposing process improvements for component
+maintenance.
+
+The term **Technical Oversight Committee** (TOC) will mean the group
+responsible for the overall oversight and stewardship of the Open edX
+project. The TOC itself was established and is governed by a formal
+charter.  You can read the full document `here
+<https://openedx.org/wp-content/uploads/2021/12/TCRIL-Technical-Oversight-Committee-Charter-rev.11-16.21.pdf>`_.
+
+Historical Context
+------------------
+Historically, the Open edX project has not had a well-defined role
+of maintainer.  At times, community members stepped into this role
+informally.  The role was also part of a broader "owner" role at edX.
+Owners were also responsible for the health of the component they
+owned both in the repository and in production; owners answered the
+pager when something went wrong.
+
+What Maintainers Do
+-------------------
+The role of the repository or project maintainer is both a community
+focused and a technical role[#]_.  Maintainers will be the public face
+of each component and should be welcoming and available.
+
+Non-technical responsibilities of maintainers include:
+
+* Effective communication with community members, other maintainers,
+  and project leadership
+* Disciplined management to ensure that maintenance work is done
+* Enforcement of the project's standards
+* The ability to solve or escalate issues facing users of the
+  component
+
+Technical responsibilities of maintainers include:
+
+* Understanding the current component architecture and implementation
+* Having a technical vision for the evolution of the component
+* The ability to assess the appropriateness of proposed changes to or
+  extensions of the component
+* The role of the component in the overall platform and how it
+  interacts with other components
+
```

Comment:
Please consider, as per [this conversation on OEP-54 CCs](https://github.com/openedx/open-edx-proposals/pull/293#issuecomment-1035625823), that a maintainer responsibility might be being available to consider and approve new Core Contributors to their repo. If we think this should be a Maintainer responsibility, OEP-54 should be updated once OEP-55 becomes accepted.

Reply:
Fully agree.